tofu/rancher/custom_cluster/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "cluster_registration_token" {
+  value = data.rancher2_cluster_v2.rancher2_cluster_v2.cluster_registration_token[0]["insecure_node_command"]
```

Comment:
We only want the insecure command to be output if that variable is set to true, right? Otherwise it should be the command without the insecure flag

Reply:
hmm. Can we leave this for improvement later? Right now, the ansible scripts are setup to explicitly use cert-manager self signed certs (insecure). So while I understand the ask to not hard code this to the insecure command, for now I feel it makes the most sense to leave it as insecure. Also having it not hard coded will make getting the correct output a little more challenging.

Reply:
Sure, I created a task so that it doesn't get lost: https://github.com/rancher/qa-tasks/issues/1909